src/Chapter1.hs

```diff
@@ -359,301 +277,250 @@ scripts together.
 the indentation rules when your body exceeds the definition line.
-
 The same function body can be written on a separate line, minding the
 indentation.
-
 @
 roundSubtract x y =
     ceiling x - y
 @
-
 Putting everything together, the complete function definition looks like this:
-
 @
 roundSubtract :: Double -> Int -> Int
 roundSubtract x y = ceiling x - y
 @
-
 Now you are ready for defining your own functions!
 -}
 
 {- |
 In our training, for some functions types are provided for you. For others, you
 need to write types manually to challenge yourself.
-
 Don't forget the main rule:
 **Always provide type signatures for top-level functions in Haskell.**
 -}
 
 
 {- |
 =⚔️= Task 3
-
 Below you see the function that finds a square of the sum of two integers. Your
 task is to specify the type of this function.
-
 >>> squareSum 3 4
 49
 -}
-
+squareSum :: Int -> Int -> Int
 squareSum x y = (x + y) * (x + y)
 
 
 {- |
 =⚔️= Task 4
-
 Implement the function that takes an integer value and returns the next 'Int'.
-
 >>> next 10
 11
 >>> next (-4)
 -3
-
 ♫ NOTE: The current function body is defined using a special function called
   "error". Don't panic, it is not broken. 'error' is like a placeholder, that
   evaluates to an exception if you try evaluating it. And it also magically fits
   every type ｡.☆.*｡. No need to worry much about "error" here, just replace the
   function body with the proper implementation.
 -}
 next :: Int -> Int
-next x = error "next: not implemented!"
+next x = x + 1
 
 {- |
 After you've implemented the function (or even during the implementation), you
 can run it in GHCi with your input. To do so, first, you need to load the module
 with the function using the ":l" (short for ":load") command.
-
 ghci> :l src/Chapter1.hs
-
 After that, you can call the 'next' function as you already know how to do that.
 Or any other function defined in this module! But remember, that you need to
 reload the module again after you change the file's content. You can reload the
 last loaded module by merely typing the ":r" command (no need to specify the
 name again).
-
 ghci> :r
-
 A typical workflow looks like this: you load the module once using the ":l"
 command, and then you should reload it using the ":r" command each time you
 change it and want to check your changes.
 -}
 
 {- |
 =⚔️= Task 5
-
 Implement a function that returns the last digit of a given number.
-
 >>> lastDigit 42
 2
-
 🕯 HINT: use the `mod` function
-
 ♫ NOTE: You can discover possible functions to use via Hoogle:
     https://hoogle.haskell.org/
-
   Hoogle lets you search Haskell functions either by name or by type. You can
   enter the type you expect a function to have, and Hoogle will output relevant
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod n 10
```

Comment:
Your implementation is almost correct :ok: 
Unfortunately, it returns negative numbers on negative inputs because of how `mod` works. Sometimes corner cases can be tricky to spot and fix...